src/app/hooks/useClickTrackerHandler/README.md

```diff
@@ -0,0 +1,130 @@
+# useClickTracker hook
```

Comment:
Should we add a note here to say that the click tracker should always be used in the context of tracked view? I agree with the components themselves not being combined together but it is important that engineers know that a click isn't very useful to Product without a view I think?

Reply:
@andrewscfc does this look OK? https://github.com/bbc/simorgh/pull/9101/commits/1db6ac77035d4d98a16e8c8881d355bca33062ff#diff-fb7721b3aeed707492606afb9ed4233e97cbbe722ffb629f0c0bb7428522e7ac